src/apps/components/juttle-editor.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import AceEditor from 'juttle-react-ace';
+
+import 'brace/theme/monokai';
+import './juttle-mode/mode';
+
+class JuttleEditor extends React.Component {
+    render() {
+        var program = this.props.bundle ? this.props.bundle.program : '';
+        var RUN_JUTTLE = {
+            name: 'RUN_JUTTLE',
+            bindKey: {win: 'Ctrl-Enter',  mac: 'Command-Enter'},
+            exec: this.props.runJuttle
+        };
+        var commands = [RUN_JUTTLE];
+        return (
+            <AceEditor
+              theme="monokai"
+              height="200"
```

Comment:
can we make the editor expand to the length of the buffer? You can see how we did it in jut-io/product: https://github.com/jut-io/product/blob/master/src/applib/components/editor/editor.jsx